modules/setting/setting.go

```diff
@@ -1078,8 +1081,6 @@ func NewContext() {
 		log.Fatal(4, "Failed to map Cron settings: %v", err)
 	} else if err = Cfg.Section("git").MapTo(&Git); err != nil {
 		log.Fatal(4, "Failed to map Git settings: %v", err)
-	} else if err = Cfg.Section("api").MapTo(&API); err != nil {
```

Comment:
This can't be removed

Reply:
Why? It has been replaced with this:
```
sec = Cfg.Section("api")
API.EnableSwaggerEndpoint = sec.Key("ENABLE_SWAGGER_ENDPOINT").MustBool(true)
API.MaxResponseItems = sec.Key("MAX_RESPONSE_ITEMS").MustInt(50)
```

Reply:
But that is not exactly the same as this will not keep defaults if added later for API struct

Reply:
I have changed it the way you want, but could you explain me a bit more what exactly did you mean by saying

> this will not keep defaults if added later for API struct

?

Reply:
Sorry, that sentence was totally messed up :) I meant that if new option is added it will not be autoloaded without adding code to load it explicitly like it is for other sections if defaults are not needed

Reply:
No problem. ;) I get it now. On the other hand, ini handling will be more prone to unexpected refactoring side effects (eg. changing struct field name will restore default value for existing deployments), but as usual, it's a matter of compromise and priorities. ;)